internal/collect/datacenters.go

```diff
@@ -38,10 +39,15 @@ func Datacenters(config *load.Config) {
 		}
 	}
 
+	cs, err := newCacheStore(config)
+	if err != nil {
+		config.Logrus.WithError(err).Warn("could not create cache for vsphere events. all events will be returned")
```

Comment:
I am not sure that the message reflect what is happening in case of error. 
Has far as I understand cs is nil, would we risk a nil pointer exception?

Reply:
Yes you're right. Before I was returning an in-memory store, then i changed the code slightly and removed it. I should add it again